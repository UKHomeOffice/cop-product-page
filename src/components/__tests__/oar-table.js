import React from "react"
import renderer from "react-test-renderer"
import OarTable from "../oar-table"

describe("OarTable", () => {

  it("renders correctly in content", () => {
    const tree = renderer
      .create(<OarTable table={[
        { rowHeader: 'Cyclamen', rowContent: '&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;'}]}/>)
      .toJSON();
    expect(tree).toMatchSnapshot()
  })
})