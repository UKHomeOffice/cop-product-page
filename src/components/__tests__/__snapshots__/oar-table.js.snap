// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OarTable renders correctly in content 1`] = `
<div>
  <div
    className="govuk-grid-row displayFlexOnDesktop"
    style={
      Object {
        "marginBottom": "2px",
      }
    }
  >
    <div
      className="govuk-grid-column-one-third"
      style={
        Object {
          "backgroundColor": "rgba(0, 53, 168, 0.69)",
        }
      }
    >
      <div
        className="govuk-body"
        style={
          Object {
            "color": "white",
            "marginBottom": "auto",
            "marginTop": "auto",
            "padding": "10px",
            "textAlign": "left",
          }
        }
      >
        Cyclamen
      </div>
    </div>
    <div
      className="govuk-grid-column-two-thirds"
      style={
        Object {
          "backgroundColor": "#eef2f9",
        }
      }
    >
      <div
        className="govuk-body"
        dangerouslySetInnerHTML={
          Object {
            "__html": "&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;",
          }
        }
        style={
          Object {
            "marginBottom": "auto",
            "marginTop": "auto",
            "padding": "10px",
            "textAlign": "left",
          }
        }
      />
    </div>
  </div>
</div>
`;
